docs/porting.md

```diff
@@ -62,6 +62,8 @@ All of the code for the low-level device API is in `src/portable/<vendor>/<chip
 ##### dcd_init
 Initializes the USB peripheral for device mode and enables it.
 
+If the MCU has an internal pull-up, this function should leave it disabled.
```

Comment:
as note: dcd doesn't need to explicitly disable PU if it is enabled by default on power/reset

Reply:
Modified.